lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_nas_procedures.c

```diff
@@ -76,6 +76,8 @@
 #include "lte/gateway/c/core/oai/tasks/s1ap/s1ap_common.h"
 #include "orc8r/gateway/c/common/service303/includes/MetricsHelpers.h"
 
+#define EXT_UE_AMBR_UL 1000000000
```

Comment:
Currently these macros used only in one function, can we move these declaration to function?

Reply:
@rsarwad IMO even if they are used as of now on only one function, using the macros here or constants help to describe the usage of them instead of the numbers